### DISCUSSION TITLE
Dark mode support

### DISCUSSION THREAD

**Discussion by robfrye** [SamDel/ChromeCast-Desktop-Audio-Streamer#122] [Ideas]
Hey there, thank you so much for making this app! It's the missing piece in my DIY whole home vinyl streaming setup (turntable through Behringer USB audio interface to windows laptop, then using this app streaming through Chromecast audio devices around the house).

One quesion: Any chance of adding Dark Mode support? The bright white window is pretty glaring at night

**Comment by SamDel** [comment]
Thanks!
In [Setup 3.8.6.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/7658396/Setup.3.8.6.zip) I added an option to enable dark mode. How's that for you?

**Comment by robfrye** [comment]
Wow, that was fast. Much better, thanks!
